### ISSUE TITLE
handling HTTP redirect for AMI IDs

### ISSUE THREAD

**Issue by mdahlhausen** [NREL/OpenStudio-aws-gem#10]
cloning and running a template openstudio-analysis-spreadsheet analysis gives the error: "302 Unable to download AMI IDs".  It seems that the `retrieve_json` call in the `ami_list.rb` script in the `openstudio-aws-gem` is not able to access the json file with AMI IDs.  It sends a get request, gets a 302 error, and instead of redirecting, gives an error. 

I worked around this by adding this just after the get request: 
```ruby
if resp.code == '302'
  resp = Net::HTTP.get_response(URI(resp['location']))
end
```	
there probably is a more elegant way to handle redirection responses

**Comment by nllong** [comment]
Thanks @mdahlhausen. I fixed this in [v0.3.1](https://github.com/NREL/OpenStudio-aws-gem/blob/020573eeaf28bed81e292575d94b5381a59b42c4/lib/openstudio/lib/ami_list.rb#L118-L145). This happened because the website that was hosting the ami lists moved to a new location and they decided to use redirects. 

try `bundle update` if you are using the spreadsheet (make sure that the Gemfile has ~> 0.3 constraint on  the openstudio-aws gem

**Comment by mdahlhausen** [comment]
Thanks @nllong for the fix; the most recent version works for me.
